### DISCUSSION TITLE
ValueError: Found input variables with inconsistent numbers of samples: [964, 2650]

### DISCUSSION THREAD

**Discussion by PhD-GOAT** [raphaelvallat/yasa#60] [Q&A] [answered]
`from sklearn.metrics import accuracy_score

---> print(f"The accuracy is {100 * accuracy_score(hypno, hypno_pred):.3f}%")`

I did it with the edf file, which is open data, and there is an error.
The channels are ['EEG Fpz-Cz', 'EEG Pz-Oz', 'EOG horizontal', 'Resp oro-nasal', 'EMG submental', 'Temp rectal', 'Event marker'].

**Comment by raphaelvallat** [comment, marked as answer]
Hi @PhD-GOAT,

The `hypno_pred` variable has a 30-sec resolution, i.e. one value per 30-sec of data. I think that the resolution of your `hypno` variable is probably not 30-seconds, but shorter (5, 10 or 15-seconds) which explains why the sizes of the two arrays are not the same.

**Reply by PhD-GOAT** [reply]
Hello!! @raphaelvallat

yasa.sleep_statistics(hypno, sf_hyp=1/30)

I didn't touch the frequency in your Yasa example. The other is the edf file on physionet.
